src/commands/docker.test.ts

```diff
@@ -192,3 +194,61 @@ describe("commands/docker.ts orderWorkflows()", () => {
     assert.strictEqual(orderedWorkflows[0].resourceKind, stubSchemaRegistryWorkflow.resourceKind);
   });
 });
+
+describe.only("commands/docker.ts addDockerPath()", () => {
```

Comment:
We'll need to make sure the rest of the tests run:
```suggestion
describe("commands/docker.ts addDockerPath()", () => {
```